cmd/combine/combine.go

```diff
@@ -113,7 +113,8 @@ func Combine(ctx context.Context, inputDir, outputDir string, force bool, opts .
 		}
 
 		if valPk != genPubkey {
-			return errors.New("generated and lockfile public key for validator DO NOT match", z.Int("validator_index", idx))
+			return errors.New("unexpected resulting combined validator public key",
+				z.Int("validator_index", idx), z.Hex("actual", genPubkey[:]), z.Hex("expected", valPk[:]))
```

Comment:
I would still keep expected and actual pubkeys, helps with debugging.

Reply:
indeed, I added them here, they weren't there previously.